ComfortZoneApp/app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +36,20 @@
         <activity
             android:name=".HostActivity"
             android:exported="false" />
+
+        <uses-library
+            android:name="org.apache.http.legacy"
+            android:required="false" />
+
+        <meta-data
+            tools:replace="android:value"
+            android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyAACbsqNaCEJh7gqU1bfSQvGuiqqOcFJlI" />
```

Comment:
this should help! https://stackoverflow.com/questions/28954071/how-can-i-access-a-buildconfig-value-in-my-androidmanifest-xml-file